marlowe-cli/test/double-satisfaction.sh

```diff
@@ -334,8 +343,8 @@ if cardano-cli transaction build --testnet-magic "$MAGIC" --alonzo-era
                                  --invalid-hereafter "$TIMEOUT_SLOT"          \
                                  --out-file tx-2.raw
 then
-  echo "Double satisfaction was prevented."
+  echo "Failure! Double satisfaction occurred"
   exit 1
 else
-  echo "Double satisfaction was not prevented."
+  echo "Success. Double satisfaction was prevented"
```

Comment:
@bwbush Just double checking that this edit is correct - if the command succeeds it means that double satisfaction occurs and that the test should fail. This check is the opposite of what it was before - was it wrong before?

Reply:
The message was wrong, but the exit code was correct. The `marlowe-cli/run-nonpab-tests.sh` correctly reported success or failure.